src/main/java/uk/gov/justice/digital/nomis/jpa/entity/CaseloadAgencyLocation.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.digital.nomis.jpa.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "CASELOAD_AGENCY_LOCATIONS")
+@Data()
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
+@EqualsAndHashCode(of = {"caseload", "agencyLocation"})
+@ToString(of = {"caseload", "agencyLocation", "updateAllowedFlag"})
+public class CaseloadAgencyLocation implements Serializable {
+
+    @EmbeddedId
+    private CaseloadAgencyIdentity id;
+
+    @OneToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "CASELOAD_ID", updatable = false, insertable = false)
```

Comment:
I don't think updatable and insertable are necessary where we are not also mapping the column?

Reply:
We are mapping the column in the ID